styleguide/derek/assets/icon-font.jade

```diff
@@ -0,0 +1,1427 @@
+    .col-md-12
+      h4.red.no-margin Rackspace
+      h2.no-margin Iconography
+  .row
+    .col-md-8
+      p Our icons help us to express the attributes of our brand, products, services, workloads, technology and ideas in ways that are immediately recognizable.
+      p We use icons as a graphical representation of a product, service, workload, technology or idea. They should not stand alone. Instead, they should be accompanied by a headline or body copy, and the corporate logo should appear somewhere on the same page or asset.
+
+  hr
+
+  .row.half-padding-full
+    .col-md-8
+      h5.red Rules
+      ul
+        li Icons are one color and used in dark, medium and light gray.
+        li An icon should be used in the context of an idea, service offering, solution area or workload where possible (paired with solution area/workload names, part of an ideogram, diagram or illustration).
+        li Make sure that all icons used together have the same line weight.
+  hr
+
+  .row
+    .col-md-12
+      h5.red Usage:
+      p In order to use the Rackspace web font in your markup you will need to put in <code>&#60;i class="rsweb name-of-icon">&#60;/i></code> replacing "name-of-icon" with the name that appears underneath the icon you've chosen.
+      p To change the font color to white add the class "white" to your icon class. <br/>
+      h6 Example
+      code &#60;i class="rsweb name-of-icon white">
+  .row.half-padding
+    .col-md-6
+      h4 Downloads
+      ul
+        li
+          a(href="fonts/rsweb.ttf") Font Files
+        li
+          a(href="icons/icons.zip") svgs
+        li
+          a(href="icons/iconpack.json") JSON file
+
+  .row
+    .col-md-12
+      h5.red Available Icons:
+
+.bg-light-gray.standard-padding-full
+    ul.nav.nav-tabs(role="tablist")
+      li.active(role="presentation")
+        a(href="#cloud-products", aria-controls="cloud-products", role="tab", data-toggle="tab") Cloud Products
+      li(role="presentation")
+        a(href="#dedicated-products", aria-controls="dedicated-products", role="tab", data-toggle="tab") Dedicated Products
+      li(role="presentation")
+        a(href="#managed", aria-controls="managed", role="tab", data-toggle="tab") Managed Services
+      li(role="presentation")
+        a(href="#auto", aria-controls="auto", role="tab", data-toggle="tab") Automated Services
+      li(role="presentation")
+        a(href="#services", aria-controls="auto", role="tab", data-toggle="tab") Services
+      li(role="presentation")
+        a(href="#general", aria-controls="auto", role="tab", data-toggle="tab") General
+      li(role="presentation")
+        a(href="#idiograms", aria-controls="auto", role="tab", data-toggle="tab") Idiograms
+
+    .tab-content
+      .container
+        #cloud-products.tab-pane.active(role="tabpanel")
+          .row
+            .col-md-3.icon-grid-item
+              .col-sm-6.center-block
+                <i class="rsweb cl-autoscale"></i>
```

Comment:
I talked to @bighappyface about this one. I'm having trouble getting jade to cooperate when adding html elements in line. ie. Icons, breaks, code tags ect. I've tried everything I could find on the google and it just is not having it. If someone doesnt mind helping me out with getting that ironed out that would be a huge help!